rules/cross-platform/persistence_shell_profile_modification.toml

```diff
@@ -0,0 +1,73 @@
+[metadata]
+creation_date = "2021/01/19"
+maturity = "production"
+updated_date = "2021/01/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may establish persistence by executing malicious content triggered by a user’s shell. ~/.bash_profile and
+~/.bashrc are shell scripts that contain shell commands. These files are executed in a user's context when a new shell
+opens or when a user logs in so that their environment is set correctly.
```

Comment:
```suggestion
Both ~/.bash_profile and ~/.bashrc are files containing shell commands that are run when Bash is invoked. These files are executed in a user's context, either interactively or non-interactively, when a user logs in so that their environment is set correctly. Adversaries may abuse this to establish persistence by executing malicious content triggered by a user’s shell.
```

Reply:
re-lint